app/controllers/user_posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UserPostsController < ApplicationController
+  def index; end
+
+  def show; end
+end
```

Comment:
- [ ] [Optional] I would strongly suggested that you rename this file from `user_posts_controller.rb` to `posts_controller.rb`. This way there will be uniformity with the model and even database table name and you will be following the rails convention. Also remember to make same adjustments to views and routes.

Reply:
hi, @oloomoses thank you for the suggestions, I'll fix it before merging it.